### ISSUE TITLE
question

### ISSUE THREAD

**Issue by amongussusususu** [hackerb9/gwordlist#3]
im trying to execute the .sh file but i cant can i have the full file so i can download it

**Comment by hackerb9** [comment]
Yes. https://github.com/hackerb9/gwordlist/raw/master/1gramsbyfreq.txt.gz

**Comment by amongussusususu** [comment]
is that the biggest version without removed words?

**Comment by hackerb9** [comment]
No. For all 29 million "English words" that Google's corpus contains, please see

https://github.com/hackerb9/gwordlist/raw/master/frequency-all.txt.gz

Note that Google only published entries which appeared at least 40 times. 

I should warn you that there are not 29 million different English words. Even the unabridged Oxford English Dictionary contains less than a quarter million words. Which means the "all" list is 99% tripe. You'll find OCR errors like the word "IIIIIIIIIIIIIIIIIIIIIIIIIIIII" (spelled with up to 75 <kbd>I</kbd>s!) and email addresses for the various journals they scanned. 

I prefer to use the lists that have been filtered either by dictionary or frequency count. As it happens, when I removed all entries not found in any of three different online dictionaries, I ended up with about [a quarter million words](https://github.com/hackerb9/gwordlist/raw/master/frequency-alpha-alldicts.txt), which seems about right. 

If you find a better way to prune out the garbage words, or if there is something missing from my shortened lists, please let me know.

**Comment by hackerb9** [comment]
P.S. I fixed the shell script so it is not in github's LFS any more so it should run on your machine easily.
